### DISCUSSION TITLE
DICOMWeb / WADO endpoints

### DISCUSSION THREAD

**Discussion by codyebberson** [medplum/medplum#3749] [General]
Placeholder discussion for gathering resources on DICOMWeb / WADO support

* https://github.com/dcmjs-org/dicomweb-server
* https://github.com/dcmjs-org/dcmjs
* https://github.com/RadicalImaging/ohif-aws-healthimaging
* https://github.com/OHIF/Viewers
* https://github.com/PantelisGeorgiadis/dcmjs-dimse

**Comment by marcoadasilvaa** [comment]
Hello, I’m using dcmjs-imaging as a lightweight viewer for DICOM-format images
